client/hotrod-client/src/test/java/org/infinispan/client/hotrod/event/ClientListenerWithFilterAndProtobufTest.java

```diff
@@ -48,24 +50,25 @@
 
    @Override
    protected void createCacheManagers() throws Throwable {
-      ConfigurationBuilder cfgBuilder = hotRodCacheConfiguration(getDefaultClusteredCacheConfig(CacheMode.DIST_SYNC, false));
+      ConfigurationBuilder defaultClusteredCacheConfig = getDefaultClusteredCacheConfig(CacheMode.DIST_SYNC, false);
+      defaultClusteredCacheConfig.encoding().key().mediaType(MediaType.APPLICATION_PROTOSTREAM_TYPE);
+      defaultClusteredCacheConfig.encoding().value().mediaType(MediaType.APPLICATION_PROTOSTREAM_TYPE);
+      ConfigurationBuilder cfgBuilder = hotRodCacheConfiguration(defaultClusteredCacheConfig);
       createHotRodServers(NUM_NODES, cfgBuilder);
       waitForClusterToForm();
 
-      for (int i = 0; i < NUM_NODES; i++) {
-         server(i).addCacheEventFilterFactory("custom-filter-factory", new CustomCacheEventFilterFactory());
-
-         // WARNING! This is not the actual instance used at runtime. A new instance is created instead, so all state is lost unless it is re-created in the constructor!
-         ProtoStreamMarshaller marshaller = new CustomProtoStreamMarshaller();
```

Comment:
CustomProtoStreamMarshaller class is no longer used here but it still exists in ClientListenerWithFilterAndProtobufTest. Please remove the dead code.